src/amo/components/Collection/index.js

```diff
@@ -71,11 +75,18 @@ export class CollectionBase extends React.Component<Props> {
   }
 
   loadDataIfNeeded(nextProps?: Props) {
-    const { collection, errorHandler, loading, params } = {
+    const { creating, collection, dispatch, errorHandler, isLoggedIn, loading, params } = {
       ...this.props,
       ...nextProps,
     };
 
+    if (creating) {
+      if (isLoggedIn) {
+        dispatch(openFormOverlay(CREATE_COLLECTION_OVERLAY));
```

Comment:
The approach I took for locating this page was to have an unloaded collection appear in the background as this logic (for creating a collection) seems to make sense to have in the `Collection` component. I'm not totally happy with it, and there's an issue that if you cancel the overlay you are left at an unloaded collection screen, so advice here would be appreciated.